llvm/test/CodeGen/RISCV/stack-guard-global.ll

```diff
@@ -0,0 +1,47 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=riscv64-unknown-elf -verify-machineinstrs < %s \
+; RUN:     | FileCheck %s
+
+define void @foo(i64 %t) sspstrong {
```

Comment:
What about the nounwind? Did you decide against that, or just miss it?

Reply:
You were right, it doesn't need nounwind; we're just testing the stack protector code here.

Reply:
No as in I would suggest adding it so that you don't get CFI being tested. uwtable was what was removed (which _forces_ unwind info to be emitted), but because this function calls `@baz`, which might throw an exception, it still ends up emitting unwind info, unless you add `nounwind` here to tell it that it can assume it's not needed.